### PR TITLE
Update build workflow to go 1.18

### PR DESCRIPTION
**Pull request by lumjjb**
Signed-off-by: Brandon Lum <lumjjb@gmail.com>

### CODE REVIEW

**Review by puerco — APPROVED** [review]
I have no juice here but LGTM

**Review by swinslow — APPROVED** [review]

**Comment by swinslow** [comment]
Hmm -- I'm fine to merge this, but for some reason it seems stuck, waiting for the tests to run but also not showing them as a check, yet blocking merging until it completes...



I'll see about undoing the block on merging without checks passing b/c it looks like something is stuck here.

**Comment by swinslow** [comment]
Closing and then re-opening the PR seems to have fixed it.  :shrug:  Merging now!

**Pull request merged**
